src/Sarif.Viewer.VisualStudio/ErrorList/StubDisposable.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Sarif.Viewer.ErrorList
+{
+    // This class exists solely to provide an object for the "stub" table data sources to return
+    // when the TableDataManager calls Subscribe on them.
+    internal class StubDisposable : IDisposable
```

Comment:
>StubDisposable [](start = 19, length = 14)

looks like you don't require more than a single static instance of this class?

Reply:
I suppose, since `Dispose` doesn't actually do anything -- but I really don't want a Singleton `IDisposable` which in fact will be disposed multiple times by VS (once per stub data source). And since there are only ever two such stub data sources, I'd rather not change this.

---
In reply to: [538766265](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538766265) [](ancestors = 538766265)